### DISCUSSION TITLE
In an MDX file, can I use frontmatter instead of metadata?

### DISCUSSION THREAD

**Discussion by shifenhutu** [vercel/next.js#69843] [Help]
### Summary

in mdx file , i have to write two times  **title、description ** , 

 the  `export const metadata = { ... }` is for seo( search engine optimization),   can I use frontmatter instead of metadata? 

**page.mdx**
```
---
title: this is my title
description: this is my description
published: "2023-11-11"

---

export const metadata = {
  title: "this is my title",
  description: `this is my description`,
};


```

**Comment by Shriharsh-Deshmukh** [comment]
Yes, you can use **frontmatter** in MDX for metadata like title, description, etc., but **SEO-specific metadata** might still need to be handled separately. Some frameworks (like Next.js) require `export const metadata` for SEO purposes. So, frontmatter simplifies things, but you might need both in cases where the framework relies on `metadata` for SEO.
